scripts/CinnabarGym.asm

```diff
@@ -135,13 +135,14 @@ CinnabarGymScript2:
 	ld [wCurMapScript], a
 	ret
 
-CinnabarGymScript3:
+CinnabarGymBlainePostBattle:
 	ld a, [wIsInBattle]
 	cp $ff
-	jp z, CinnabarGymScript_75792
+	jp z, CinnabarGymResetScripts
 	ld a, $f0
 	ld [wJoyIgnore], a
-CinnabarGymScript3_75857:
+
```

Comment:
The lack of blank line was to indicate the fallthrough nature of this routine. I think it would be better to either delete this blank line or add a `; fallthrough` comment.

Reply:
I like explicit `; fallthrough` comments.

Reply:
@KqesaR There are a few more instances of this. Otherwise, looks good!